internal/util/importutilv2/parquet/util.go

```diff
@@ -274,7 +274,18 @@ func isSchemaEqual(schema *schemapb.CollectionSchema, arrSchema *arrow.Schema) e
 // todo(smellthemoon): use byte to store valid_data
 func bytesToBoolArray(length int, bytes []byte) []bool {
```

Comment:
The function name bytesToBoolArray is too broad
 append True is bytes is empty is only appliable to store valid_data?

Reply:
"bytes" is a byte array obtained from parquet file, each bit represents whether a row is null or not.
bytesToBoolArray() is to convert the byte array to a bool array, which is required by storage.InsertData.AppendRows()